### PR TITLE
Added Number OF times Sorted array rotated (using binary Search) in cpp

### PR DESCRIPTION
**Pull request by bitaashna**
I have added **'Number of times sorted array rotated'** in **Cpp**
#169

### CODE REVIEW

**Comment by OjusWiZard** [comment]
Add the Algo name in the readme

**Comment by bitaashna** [comment]
@OjusWiZard Done!

File: Code/C++/no_of_rotation.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    //if last element is greater than 1st that is array sorted
+    //return 0 as number of rotation equals zero
+    if (arr[0] < arr[n - 1]) {
+        return 0;
+    }
+
+    //declaring and initializing two pointers l and r
+    //pointing to the leftmost and rightmost element respectively.
+    int l = 0, r = n - 1;
+    //iterating the elements until i is less than equal to r
```
Comment (OjusWiZard) [inline Code/C++/no_of_rotation.cpp:28]: 'l' instead of i

File: Code/C++/no_of_rotation.cpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	
+  * Input :-
+    Enter size of array: 6
+    Enter array Elements: 60 70 80 20 40 50
+    
+  * Output :-
+    Number of times sorted array is rotated: 3
+
+  * Time complexity: O(n)
+  * Space Complexity: O(n)
```
Comment (OjusWiZard) [inline Code/C++/no_of_rotation.cpp:91]: Complexities seem incorrect to me. Please recheck.

File: Algorithm/DP/readme.md [hunk 3]
```diff
@@ -41,4 +41,7 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 - Longest Common Subsequence ----> [C++](/Code/C++/longest_common_subsequence.cpp)
 
+- Number of times sorted array rotated ---->[C++](/Code/C++/no_of_rotation.cpp)
+
```
Comment (OjusWiZard) [inline Algorithm/DP/readme.md:45]: There is no Dynamic Programming approach in this algorithm. So, please don't link in the DP folder.

**Review by OjusWiZard — CHANGES_REQUESTED** [review]

**Comment by bitaashna** [comment]
@OjusWiZard Please review,I have made the changes

**Review by OjusWiZard — APPROVED** [review]

**Review by nakul-19 — APPROVED** [review]

**Pull request merged**